src/account/AccountBalanceQuery.ts

```diff
@@ -21,8 +22,15 @@ export class AccountBalanceQuery extends QueryBuilder<Hbar> {
         this._inner.setCryptogetaccountbalance(this._builder);
     }
 
-    public setAccountId(accountId: AccountIdLike): this {
-        this._builder.setAccountid(new AccountId(accountId)._toProto());
+    public setAccountId(id: AccountIdLike): this {
+        this._builder.setAccountid(new AccountId(id)._toProto());
+        return this;
+    }
+
+    public setContractId(id: ContractIdLike): this {
```

Comment:
This is incorrect. The protobufs have to be updated. There is an actual `.setContractId` method on the proto now. 

https://github.com/hashgraph/hedera-protobuf/blob/master/src/main/proto/CryptoGetAccountBalance.proto#L17

Reply:
This requires the protobufs to be updated, so the code will be commented out for now. Plan is to update the protobufs in another PR as it isn't necessarily relevant to the current one, and would probably break quite a bit of existing code.